docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/4.delete-vertex.md

```diff
@@ -0,0 +1,19 @@
+# DELETE VERTEX
+
+```nGQL
+DELETE VERTEX <vid1>[, <vid2>...]
+```
+
+Use `DELETE VERTEX` to delete vertices and the related incoming and outgoing edges of the vertices. The `DELETE VERTEX` statement deletes one vertex or multiple vertices at a time. You can use `DELETE VERTEX` together with pipe. For more information on pipe, see [Pipe operator](../5.operators/4.pipe.md).
+
+## Examples
+
+```ngql
+nebula> DELETE VERTEX "team1";
+
+nebula> GO FROM "player100" OVER serve YIELD serve._dst AS id | DELETE VERTEX $-.id;
+```
+
+The preceding query deletes the vertex whose ID is "team1".
+
+Nebula Graph traverses the incoming and outgoing edges related to the vertices and deletes them all. Then Nebula Graph deletes information related to the vertices. Atomic operation is not guaranteed during the entire process for now, so please retry when failure occurs.
```

Comment:
"Atomic operation is not guaranteed during the entire process for now, so please retry when failure occurs." ==> "when a failure occurs" and this sentence can be a NOTE or a CAUTION?